image_enc_test.go

```diff
@@ -132,8 +136,17 @@ func TestImageEncryption(t *testing.T) {
 		},
 	}
 
+	var (
+		l leases.Lease
+	)
+
+	l, err = ls.Create(ctx, leases.WithRandomID(), leases.WithExpiration(5*time.Minute))
+	if err != nil {
+		t.Fatal("Unable to create lease for encryption")
+	}
+
```

Comment:
Should probably have the `defer` right here.

Reply:
the reason i didn't defer is because it helps us check that we didnt mess up any references. Because it deletes anything that is not properly referenced by the image.

Reply:
I realized i would have to do better error checking clean up, i will do that